overlord/snapstate/snapmgr.go

```diff
@@ -73,10 +73,9 @@ func (ssfl *SnapSetupFlags) UnmarshalJSON(b []byte) error {
 
 // SnapSetup holds the necessary snap details to perform most snap manager tasks.
 type SnapSetup struct {
-	Name     string        `json:"name"`
-	Revision snap.Revision `json:"revision,omitempty"`
-	Channel  string        `json:"channel,omitempty"`
-	UserID   int           `json:"user-id,omitempty"`
+	// FIXME: rename to RequestedChannel to convey the meaning better
```

Comment:
Yeah, I see the confusion, but the rename doesn't seem to help much. It's clear that everything there is a requested change for a given snap.